src/kibana/components/doc_viewer/doc_viewer.html

```diff
@@ -23,6 +23,16 @@
                 tooltip="Filter out value"
                 tooltip-append-to-body="1"
                 class="fa fa-search-minus"></i>
+              <i ng-click="filter(mapping[field], flattened[field],'<=')"
+                 tooltip="Filter less than or equal to value"
+                 tooltip-append-to-body="1"
+                 class="fa fa-search-less-than-equal"
```

Comment:
what properties do you get from using the class 'fa' along with your custom class?